cypress/integration/_radioButtonAlumni.spec.js

```diff
@@ -0,0 +1,13 @@
+//check radio box alumni
```

Comment:
There is no test in this file, maybe delete it. The alumni checkbox is already tested in createdCard.spec